DeskKit/DeskKit/View Models/DKContactUsViewModel.m

```diff
@@ -142,9 +162,52 @@ + (BOOL)isEmptyString:(NSString *)string
     return string == nil || [string isEqualToString:@""];
 }
 
++ (BOOL)isNotEmptyString:(NSString *)string
+{
+    return ![[self class] isEmptyString:string];
+}
```

Comment:
So would you put DK in the method names? like:
[NSString DKIsNotEmptyString:myString]

Otherwise, I don't like the idea of a SDK polluting the global namespace of an app with traditionally named category methods.